src/cephadm/cephadm.py

```diff
@@ -913,6 +913,10 @@ def create_daemon_dirs(self, data_dir, uid, gid):
         # populate files from the config-json
         populate_files(data_dir, self.files, uid, gid)
 
+        if isinstance(self.ctx.container_engine, Podman):
+            if os.path.exists('/etc/hosts'):
+                shutil.copy('/etc/hosts', os.path.join(data_dir, 'hostsfile'))
```

Comment:
I guess so. I think I was worried about something the container does messing with the host's actual /etc/hosts file, but there isn't any actual examples of that I can think of so that's probably fine.